src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -1276,11 +1276,8 @@ void Transformations::MainSnippets(void) {
         },
         snippets::pass::TokenizeSnippets);
 
-    auto mm_supports_transpose_b = [this, ignoreCallback](const std::shared_ptr<const ov::Node>& n) {
+    auto mm_supports_transpose_b = [this](const std::shared_ptr<const ov::Node>& n) {
```

Comment:
Now this lambda is needed only for TPP, so I think we can cover the whole `ExplicitTransposeMatMulInputs`'s callback in `#if defined(SNIPPETS_LIBXSMM_TPP)` macros. What do you think?

Reply:
As far as I understand, we cannot do it because then for oneDNN backend [default callback](https://github.com/openvinotoolkit/openvino/blob/58b02fb88fa660468d281e751f117c51cb22932c/src/common/snippets/src/pass/explicit_transpose_matmul_inputs.cpp#L93) will return `false` and the pass `ExplicitTransposeMatMulInputs` will [decide](https://github.com/openvinotoolkit/openvino/blob/58b02fb88fa660468d281e751f117c51cb22932c/src/common/snippets/src/pass/explicit_transpose_matmul_inputs.cpp#L93) that the current `transposed_b=true` is not supported.

But I rewrote this part a little bit.

Reply:
Thanks for the correction, I already forgot how exactly we use the callback for this pass 😄